code/__DEFINES/subsystems.dm

```diff
@@ -133,6 +133,7 @@
 #define INIT_ORDER_TICKER 55
 #define INIT_ORDER_TCG 55
 #define INIT_ORDER_MAPPING 50
+#define INIT_ORDER_DAY_NIGHT 49 // Networking and atoms require this
```

Comment:
why

Reply:
Networking requires the time to be set after initialising, if the subsystem hasn't initialised, the time won't have been set to what it should be yet, resulting in runtimes and incorrect timestamps

Reply:
please add that to the comment, cheers

Reply:
donezo